habitat-lab/habitat/gym/gym_wrapper.py

```diff
@@ -178,12 +175,18 @@ class HabGymWrapper(gym.Env):
     Example usage:
     """
 
-    def __init__(self, env, save_orig_obs: bool = False):
-        gym_config = env.config.gym
-        self._gym_goal_keys = gym_config.desired_goal_keys
-        self._gym_achieved_goal_keys = gym_config.achieved_goal_keys
-        self._gym_action_keys = gym_config.action_keys
-        self._gym_obs_keys = gym_config.obs_keys
+    def __init__(
+        self,
+        env: "RLEnv",
+        habitat_gym_config: "DictConfig",
+        save_orig_obs: bool = False,
+    ):
+        super().__init__(env)
```

Comment:
gym.Wrapper.__init__ takes as input a gym.Env, but RLEnv is not a gym.Env. I am surprised mypy lets you do this, I do not think this is a good idea.

Reply:
RLEnv is a gym.Env [class RLEnv(gym.Env)](https://github.com/facebookresearch/habitat-lab/blob/1b1ec26e464cf7c490d762a1680da05449355b2b/habitat-lab/habitat/core/env.py#L357).

Reply:
My bad.